ember-browser-services/src/test-support/index.ts

```diff
@@ -105,11 +101,6 @@ export function maybeMake<
 
 // we are already using ember-window-mock, so the proxy internal to that package will
 // "just handle" setting stuff on the window
-//
-// NOTE:
-//  - Location implementation is incomplete:
-//     https://github.com/kaliber5/ember-window-mock/blob/2b8fbf581fc65e7f5455cd291497a3fdc2efdaf5/addon-test-support/-private/mock/location.js#L23
-//     - does not allow setting "origin"
```

Comment:
see the [changelog comment](https://github.com/CrowdStrike/ember-browser-services/pull/399/files#diff-b9cc623d4d04c076a2cadea0ace8062484e9e0896e98f6de7f8e79a04ac10b0cR7)